src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Emitter/CoreBindingHelpers.cs

```diff
@@ -437,7 +436,7 @@ void EmitBindImplForMember(MemberSpec member)
                         _writer.WriteLine($$"""
                             {{condition}}
                             {
-                                throw new {{Identifier.InvalidOperationException}}("{{string.Format(ExceptionMessages.ParameterHasNoMatchingConfig, type.Name, member.Name)}}");
+                                throw new {{Identifier.InvalidOperationException}}("{{string.Format(ExceptionMessages.ParameterHasNoMatchingConfig, type.FullName, member.Name)}}");
```

Comment:
>FullName [](http://example.com/codeflow?start=153&length=8)

why didn't use `type.TypeRef.FullyQualifiedName` here?

Reply:
I see now you have added FullName property but why we didn't use this in the line 357?

Reply:
For runtime exceptions I'm using the "FullName" since there are tests checking for compatibility with `System.Type.ToString()` being thrown by the reflection binder.